app.js

```diff
@@ -0,0 +1,34 @@
+let allBooks = document.querySelector('.books');
+
+document.getElementById('btn').addEventListener('click' , getBooks);
+
+
+function getBooks(){
+    // using Fetch 
+    fetch('books.json')
```

Comment:
You don't need to use fetch to display the hardcoded JSON list. Fetch can go in the next PR when getting the books for the REST API. For now it is enough just to create a JSON array of books and iterate over the array and display each of them

Reply:
Once more thank you for replying and having time for reviewing my code :)
I used f
Fetch because I'm getting books from a separate JSON file, I could use AJAX and XHR for the same purpose instead of Fetch.
Secondly I think that you mean to put an array of objects inside js file and then loop through it and show books. Right?

Reply:
That's correct